### ISSUE TITLE
examples/cpp/mnist_collection/GNUmakefile missing nnabla_utils

### ISSUE THREAD

**Issue by mu-triv** [sony/nnabla#620]
running
`cpp/mnist_collection$ make lenet`

results:

```
/usr/bin/ld: /tmp/cccYCqnu.o: in function `lenet_training_with_dynamic_graph(nbla::Context)':
train_lenet_classifier.cpp:(.text+0x3c11): undefined reference to `nbla::utils::load_parameters(nbla::ParameterDirectory&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >)'
/usr/bin/ld: train_lenet_classifier.cpp:(.text+0x5b3a): undefined reference to `nbla::utils::save_parameters(nbla::ParameterDirectory&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >)'
/usr/bin/ld: /tmp/cccYCqnu.o: in function `lenet_training_with_static_graph(nbla::Context)':
train_lenet_classifier.cpp:(.text+0x6876): undefined reference to `nbla::utils::load_parameters(nbla::ParameterDirectory&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >)'
/usr/bin/ld: train_lenet_classifier.cpp:(.text+0x8473): undefined reference to `nbla::utils::save_parameters(nbla::ParameterDirectory&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >)'
collect2: error: ld returned 1 exit status
```

This is due to missing the library `nnabla_utils`.so in GNUmakefile

**Comment by TomonobuTsujikawa** [comment]
Thank you for reporting issue.
We prepared PR (https://github.com/sony/nnabla-ext-cuda/pull/270), and was already released in v1.16.0.

**Comment by TomonobuTsujikawa** [comment]
Close this issue.
Please let us know if you notice any other issues.
